### PR TITLE
login sign up

### PR DESCRIPTION
**Pull request by Abhishek-Shr**

### CODE REVIEW

File: client/src/components/Filter/Filter.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```
Comment (abhinav941) [inline client/src/components/Filter/Filter.js:1]: delete this

File: client/src/components/Login/index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { Button } from 'antd';
+import styled from "styled-components";
```
Comment (abhinav941) [inline client/src/components/Login/index.js:3]: fix import order

Comment (abhinav941) [reply client/src/components/Login/index.js:3]: default import
one line space
named import 
one line space 
code.....

File: client/src/components/ProductStack/ProductItem/index.js [hunk 3]
```diff
@@ -1,24 +1,31 @@
 import React from 'react';
 // import './ProductItem.css';
```
Comment (abhinav941) [inline client/src/components/ProductStack/ProductItem/index.js:2]: remove if not used

Comment (abhinav941) [reply client/src/components/ProductStack/ProductItem/index.js:2]: import order to be fixed

File: client/src/components/Registration/index.js [hunk 4]
```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Button } from 'antd';
+import {GooglePlusCircleFilled,FacebookFilled,TwitterCircleFilled,LinkedinOutlined  } from "@ant-design/icons";
+// import {  } from "@ant-design/icons";
```
Comment (abhinav941) [inline client/src/components/Registration/index.js:5]: remove this

**Review by abhinav941 — CHANGES_REQUESTED** [review]

**Comment by Abhishek-Shr** [comment]
Aye sir !!!!

**Comment by Abhishek-Shr** [comment]
Review changes done!! Sent from Mail for Windows 10 From: AbhinavSent: Monday, July 27, 2020 11:40 PMTo: abhinav941/e-commernceCc: Abhishek-Shr; AuthorSubject: Re: [abhinav941/e-commernce] login sign up (#7) @abhinav941 requested changes on this pull request.In client/src/components/Filter/Filter.js:> @@ -0,0 +1,11 @@+import React from 'react';delete thisIn client/src/components/Login/index.js:> @@ -0,0 +1,118 @@+import React from 'react';+import { Button } from 'antd';+import styled from "styled-components";fix import orderIn client/src/components/Login/index.js:> @@ -0,0 +1,118 @@+import React from 'react';+import { Button } from 'antd';+import styled from "styled-components";default importone line spacenamed importone line spacecode.....In client/src/components/ProductStack/ProductItem/index.js:> @@ -1,24 +1,31 @@ import React from 'react'; // import './ProductItem.css';remove if not usedIn client/src/components/ProductStack/ProductItem/index.js:> @@ -1,24 +1,31 @@ import React from 'react'; // import './ProductItem.css';import order to be fixedIn client/src/components/Registration/index.js:> @@ -0,0 +1,136 @@+import React from 'react';+import styled from 'styled-components';+import { Button } from 'antd';+import {GooglePlusCircleFilled,FacebookFilled,TwitterCircleFilled,LinkedinOutlined  } from "@ant-design/icons";+// import {  } from "@ant-design/icons";remove this—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request merged**